guides/advanced_guides/configuration.md

```diff
@@ -39,6 +39,7 @@ Server is listening on: http://127.0.0.1:7700
 - [Payload Limit Size](/guides/advanced_guides/configuration.md#payload-limit-size)
 - [Max MDB Size](/guides/advanced_guides/configuration.md#max-mdb-size)
 - [Max UDB Size](/guides/advanced_guides/configuration.md#max-udb-size)
+- [Backup](/guides/advanced_guides/configuration.md#backup-path)
```

Comment:
Since this is way more important for the user than `Max udb size` and `Max mdb size` I would suggest you move it above Max MDB size